src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
```

Comment:
In the original code, the Ball class already overrides the toString() method and provides its own implementation. By doing this, you can directly call System.out.println(ball1.getRandomBall()) without explicitly calling toString(), because the overridden toString() method will be automatically called when you try to print the Ball object.

Therefore, having an explicit toString() method in the Ball class is redundant because the overridden toString() method already takes care of providing the string representation of the Ball object.